docs/toml-schema.md

```diff
@@ -292,4 +292,8 @@ ci-checks = ["CI"]
 # merging into this branch require another review. 
 # (optional - default `false`)
 dismiss-stale-review = false
+# How many approvals are required for a PR to be merged.
+# This option is only relevant if bors is not used.
```

Comment:
Can we enforce/validate that it is not set when bors is used?

Reply:
Sure, I'll add validation. Using bors is already quite funky though, since it's set globally per repo and we don't distinguish which branch (protection) is managed by it.

Reply:
Done.